<!DOCTYPE html>
<html lang="en">
    <head>
        <title>Feng Jiang's Homepage</title>
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
        <meta name="author" content="Feng Jiang" />
        <meta name="description" content="Homepage of Feng Jiang, PhD Student at UTA" />
        <meta name="keywords" content="Feng Jiang, PhD Student, University of Texas at Arlington" />
        <!-- Bootstrap -->
        <link href="css/bootstrap.css" rel="stylesheet" />
        <link href="css/bootstrap-responsive.css" rel="stylesheet" />
        <link href="css/main.css" rel="stylesheet" />
        <script src="js/jquery.js"></script>
        <script src="js/bootstrap.js"></script>
        <style>
            body {
                padding-top: 100px; /* Increased padding to prevent navbar overlap */
            }
            .container {
                max-width: 1200px;
                margin: 0 auto;
                padding: 0 100px; /* Add significant padding to container */
            }
            .teaching-list li {
                margin-bottom: 5px;
                line-height: 1.4;
            }
            #body {
                margin: 0 100px; /* Increased horizontal margins */
            }
            
            h4 {
                padding-top: 40px;  /* Add padding to section headings */
                margin-top: -60px;  /* Negative margin to offset padding */
                scroll-margin-top: 70px; /* Modern browsers support for scroll positioning */
            }
            
            /* Additional content padding */
            .contentItem {
                padding: 0 20px; /* Add padding to content sections */
            }
            
            /* Navbar adjustments */
            .navbar-fixed-top {
                margin-bottom: 20px;
            }
        </style>
                <style>
                    /* Previous styles remain the same */
                    
                    /* Navigation active state styles */
                    .navbar .nav > li > a {
                        color: #999;
                        transition: color 0.3s ease;
                    }
                    
                    .navbar .nav > li > a:hover {
                        color: #fff;
                    }
                    
                    .navbar .nav > .active > a,
                    .navbar .nav > .active > a:hover,
                    .navbar .nav > .active > a:focus {
                        color: #fff;
                        background-color: #333;
                    }
                </style>
        
                <script>
                    $(document).ready(function() {
                        // Initialize Bootstrap scrollspy
                        $('body').scrollspy({ 
                            target: '#navbar',
                            offset: 80
                        });
        
                        // Smooth scrolling for anchor links
                        $('a[href^="#"]').on('click', function(event) {
                            event.preventDefault();
                            var target = $(this.getAttribute('href'));
                            if(target.length) {
                                $('html, body').stop().animate({
                                    scrollTop: target.offset().top - 70
                                }, 500);
                            }
                        });
        
                        $(window).on('activate.bs.scrollspy', function() {
                            var currentItem = $('.navbar .nav > .active > a').text();
                            console.log('Current section: ' + currentItem);
                        });
                    });
                </script>
    </head>
    <body data-spy="scroll" data-target="#navbar" data-offset="80">
        <div class="container">
            <div class="span10 offset1">
                <div class="navbar navbar-fixed-top navbar-inverse" id="navbar">
                    <div class="navbar-inner">
                        <div class="container">
                            <a class="brand" href="#top">Feng Jiang</a>
                            <ul class="nav">
                                <li class="nav-item"><a href="#profile">Profile</a></li>
                                <li class="nav-item"><a href="#biography">Biography</a></li>
                                <li class="nav-item"><a href="#research">Research</a></li>
                                <li class="nav-item"><a href="#publications">Publications</a></li>
                                <li class="nav-item"><a href="#teaching">Teaching</a></li>
                                <li class="nav-item"><a href="#honors">Honors</a></li>
                                <li class="nav-item"><a href="#research_experience">Experience</a></li>
                            </ul>
                        </div>
                    </div>
                </div>
                <div id="body">
                    <section id="profile">
                    <table cellpadding="3" cellspacing="0" border="0">
                        <tbody>
                            <tr>
                                <td valign="top">
                                    <p>
                                        <h2>Feng Jiang</h2>
                                    </p>
                                    <p>
                                    PhD Student in Computer Science
                                    </p>
                                    <table width="100%" cellpadding="5" cellspacing="0" style="text-align: left;">
                                        <tbody>
                                            <tr>
                                                    University of Texas at Arlington <br/>
                                            </tr>
                                            <tr>
                                                    Arlington, TX <br/><br/>
                                            </tr>
                                            <tr>
                                                <b>Email:</b> alanfengjiang@gmail.com <br />
                                            </tr>
                                            <tr>
                                                <a href="https://scholar.google.com/citations?user=6OdM9AUAAAAJ&hl=zh-CN" target="_blank">Google Scholar</a> &nbsp;&nbsp;
                                                <a href="https://github.com/fengjiang02" target="_blank">GitHub</a> &nbsp;&nbsp;
                                                <a href="upload/Feng_Jiang_Resume.pdf" target="_blank">Download CV</a>                                            </tr>
                                            </tr>
                                        </tbody>
                                    </table>
                                </td>
                            </tr>
                        </tbody>
                    </table>
                    </section>
                    <br />
                    
                    <section id="biography">
                        <h4>Biography</h4>
                    <p class="contentItem">
                    I am a third-year PhD student in Computer Science at the University of Texas at Arlington (John S. Schuchman Outstanding Doctoral Student). My PhD advisor is <a href="https://ranger.uta.edu/~huang/" target="_blank">Prof. Junzhou Huang</a>. I received my M.S. in Software Engineering from UTA and B.E. in Software Engineering from Xi'an Jiaotong University City College.
                    </p>
                    <br />
                    </section>


                    <h4>Research</h4>
                    <p class="contentItem">
                    My research interests include:
                    <br/><br/>
                    - Large Foundation Models & Multimodal Large Language Models<br/>
                    - Graph Neural Networks<br/>
                    - Deep AUC Maximization<br/>
                    - Long-tailed Learning
                    </p>
                    <br />
                    </section>


                    <section id="publications">
                    <h4>Publications</h4>
                    <ol type="1">
                        <li><b>Feng Jiang</b>, et al. <a href="https://academic.oup.com/bib/article/25/4/bbae343/7713742" target="_blank">"GTE: a graph learning framework for prediction of T-cell receptors and epitopes binding specificity."</a> Briefings in Bioinformatics 25.4 (2024).</li><br />
                        
                        <li>Weizhi An, Wenliang Zhong, <b>Feng Jiang</b>, Hehuan Ma and Junzhou Huang, <a href="https://eccv.ecva.net/virtual/2024/poster/2692" target="_blank">"Causal Subgraphs and Information Bottlenecks: Redefining OOD Robustness in Graph Neural Networks"</a>, In Proc. of ECCV'24.</li><br />
                        
                        <li>Hehuan Ma, <b>Feng Jiang</b>, Yuzhi Guo and Junzhou Huang, <a href="https://dl.acm.org/doi/10.1145/3535508.3545998" target="_blank">"Towards Robust Self-training for Molecular Biology Prediction Tasks"</a>, Journal of Computational Biology, 31(3):213-228, 2024.</li><br />
                        
                        <li>Na, S., Guo, Y., <b>Feng Jiang</b>, Ma, H., & Huang, J. <a href="https://arxiv.org/abs/2401.13220" target="_blank">"Segment Any Cell: A SAM-based Auto-prompting Fine-tuning Framework for Nuclei Segmentation."</a> arXiv:2401.13220.</li><br />
                        
                        <li>Ma, H., <b>Feng Jiang</b>, Rong, Y., Guo, Y., & Huang, J. <a href="https://dl.acm.org/doi/abs/10.1145/3535508.3545998" target="_blank">"Robust self-training strategy for various molecular biology prediction tasks."</a> In Proceedings of BCB'22.</li>
                    </ol>
                    <br />
                    </section>

                    <section id="teaching">
                    <h4 id="teaching">Teaching Experience</h4>
                    <ul class="teaching-list">
                        <li>2248-CSE-6324-980-ADV TOPS SOFTWARE ENGINEERING <i>Fall 2024</i></li>
                        <li>2245-CSE-3314-002-PROFESSIONAL PRACTICES <i>Summer 2024</i></li>
                        <li>2242-CSE-6324-001-ADV TOPS SOFTWARE ENGINEERING <i>Spring 2024</i></li>
                        <li>2238-CSE-6324-980-ADV TOPS SOFTWARE ENGINEERING <i>Fall 2023</i></li>
                        <li>2232-CSE-6324-001-ADV TOPS SOFTWARE ENGINEERING <i>Spring 2023</i></li>
                        <li>2228-CSE-6324-980-ADV TOPS SOFTWARE ENGINEERING <i>Fall 2022</i></li>
                        <li>2228-CSE-5325-003-SFWR ENG II: MGMT, MAIN, & QA <i>Fall 2022</i></li>
                    </ul>
                    <br />
                    </section>

                    <section id="honors">
                    <h4 id="honors">Honors</h4>
                    <ul class="contentItem">
                        <li>John S. Schuchman Outstanding Doctoral Student Award, UTA (April 2023)</li>
                    </ul>
                    <br />
                    </section>


                    <section id="research_experience">
                    <h4 id="research_experience">Research Experience</h4>
                    <p class="contentItem">
                    <b>(2024.2-Now) Multimodal Large Language Model</b>
                    <ul>
                        <li>LLaPA: We leverage multimodal approaches using large language models to enhance protein function prediction. Our method integrates protein sequences and functional descriptions from UniProt with contrastive learning techniques to predict the potential functions of unknown sequences. The model demonstrates superior performance in protein-text retrieval tasks and downstream predictions, while also showing reliable capabilities in both protein sequence and functional description generation.</li>
                        
                        <li>AlphaEpi: We combine sequence and structure information by using the protein language model ESM-2 for sequence data and AlphaFold 3 for structural data. A dynamic alignment module merges these modalities, leading to exceptional performance in B-cell epitope prediction. <i>This work has been accepted to the ACM BCB 2024 Conference</i>.</li>
                        
                        <li>Toxmm: We developed a multi-modal molecular embedding model for toxicity prediction by integrating SMILES structures and their textual descriptions. We curated a comprehensive dataset of over 50,000 molecular SMILES-text pairs from PubChem database for pre-training. Our approach achieved state-of-the-art performance on the toxicity dataset.</li>
                    </ul>
                    </p>

                    <p class="contentItem">
                    <b>(2023.7-2024.3) A Graph Framework for TCR and Epitopes Binding Prediction</b>
                    <ul>
                        <li>Developed a novel graph-based framework that models TCR-epitope binding as a topology learning problem, utilizing residue interaction patterns and structural information to capture complex binding mechanisms.</li>
                        
                        <li>Enhanced model performance through deep AUC maximization and dynamic graph attention mechanisms, achieving state-of-the-art performance on multiple benchmark datasets.</li>
                        
                        <li>Our method significantly outperforms existing approaches in both accuracy and efficiency for TCR-epitope binding prediction. <i>Published in Briefings in Bioinformatics</i>.</li>
                    </ul>
                    </p>

                    <p class="contentItem">
                    <b>(2023.3-2024.1) SAM-based Auto-prompting Fine-tuning Framework</b>
                    <ul>
                        <li>Segment Any Cell is an innovative framework designed to enhance the Segmentation Anything Model (SAM) for nuclei segmentation.</li>
                        
                        <li>Low-Rank Adaptation (LoRA) is integrated within the attention layer of the Transformer to improve the fine-tuning process. An auto prompt generator is introduced to guide the segmentation process.</li>
                    </ul>
                    </p>

                    <p class="contentItem">
                    <b>(2022.3-2023.4) LLM in Protein Structure Prediction</b>
                    <ul>
                        <li>Developed a novel teacher-student framework to address the challenge of limited structural data in protein research. The Teacher model, trained on over 20,000 high-quality labeled structures, generates reliable pseudo labels to guide the training of a Student protein language model across millions of sequences.</li>
                        
                        <li>Published in BCB'22 and accepted by the Journal of Computational Biology 2024.</li>
                    </ul>
                    </p>
                    <br />
                    </section>

                    <div class="contentItem">
                        *Last updated on Nov 2024.
                    </div>
                </div>
            </div>
        </div>
    
        <script src="js/jquery.js"></script>
        <script src="js/bootstrap-scrollspy.js"></script>
        <script src="js/bootstrap.js"></script>

    </body>
</html>